src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,53 @@
+package duke.task;
+
+/**
+ * Implements task objects.
+ * 
+ * @author Audrey Felicio Anwar
+ */
+public class Task {
+    protected String description;
+    protected boolean done;
+
+    public Task(String description, boolean done) {
+        this.description = description;
+        this.done = done;
+    }
+
+    /**
+     * Completes a task.
+     */
+    public void completeTask() {
+        done = true;

```

Comment:
Perhaps you can standerdise the use of 'this'

Reply:
Thanks for the suggestion. However, I don't think I understand what you mean by standerdising the use of `this`. Can you elaborate further?